index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sorting-Algorithms-in-Javascript</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/css/main.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar__container wrapper">
+            <div class="navbar__logo"><a href="#" class="navbar__logo-name">sora-al.js <img src="/img/flat-color-icons_generic-sorting-asc.png" alt="" srcset=""> </a> </div>
```

Comment:
Z tego co zauważyłem w pliku `.scss`, to element `.navbar__logo` nie ma żadnych styli - może dałoby się usunąć ten div i zamiast niego zostawić sam element `a`?

Ponadto złą praktyką jest tworzenie takich jednolinijkowców - taki ciąg znaków bardzo ciężko się czyta, a z perspektywy wydajności kompletnie nic nie daje. Rozbij proszę ten fragment na kilka linijek. :)

`srcset` do wywalenia, jeśli ma pozostać puste.

`alt` powinien mieć wartość - na przykład prosty opis "Company logo".

Albo od drugiej strony: dobrze byłoby jednak wyeksportować z Figmy cały napis wraz z logo jako SVG i wtedy opisać altem np. "sora-al.js logo".